cmd/up/up.go

```diff
@@ -61,7 +61,7 @@ import (
 
 // ReconnectingMessage is the message shown when we are trying to reconnect
 const (
-	ReconnectingMessage = "Trying to reconnect to your cluster. File synchronization will automatically resume when the connection improves."
+	ReconnectingMessage = "Trying to reconnect to your context. File synchronization will automatically resume when the connection improves."
```

Comment:
```suggestion
	ReconnectingMessage = "Trying to reconnect to your cluster. File synchronization will automatically resume when the connection improves."
```
I thing in this case is clearer talk about cluster